docs/cli.md

```diff
@@ -591,3 +596,36 @@ gcli group grant --group groupA --role admin
 ```bash
 gcli group revoke  --group groupA --role admin
 ```
+
+### Topic commands
+
+#### Display a topic's details
+
+```bash
+gcli topic details --name kafka.default --topic topic3
+```
+
+#### Create a tag
+
+```bash
+gcli topic create --name kafka.default --topic topic3
+```
+
+#### List all topics
+
+```bash
+gcli topic list --name kafka.default
+```
+
+#### Delete a topic
+
+```bash
+gcli topic delete --name kafka.default --topic topic3
```

Comment:
See my unanswered question above. It can be done either way, and I don't have a strong preference, although it might be confusing to the user to equate topics and tables.